tools/onnx_subgraph/test_model_download.sh

```diff
@@ -0,0 +1,15 @@
+pip install onnx onnxsim
```

Comment:
```suggestion
#!/bin/bash

pip install onnx onnxsim
```

how about using shebang ? https://en.wikipedia.org/wiki/Shebang_(Unix)